pusher/src/Controller/BaseHttpController.ts

```diff
@@ -31,12 +32,15 @@ export class BaseHttpController {
 
         if (axios.isAxiosError(e) && e.response) {
             res.status(e.response.status);
-            res.send(
-                "An error occurred: " +
-                    e.response.status +
-                    " " +
-                    (e.response.data && e.response.data.message ? e.response.data.message : e.response.statusText)
-            );
+            const errorType = isErrorApiData.safeParse(e?.response?.data);
+            if (!errorType.success) {
+                res.send(
```

Comment:
I think in the case the parsed error is not in the correct format, you should set the status to "500".